src/twisted/test/test_tcp.py

```diff
@@ -814,7 +817,7 @@ def _check1cleanup(results, cf1):
 class MyOtherClientFactory(protocol.ClientFactory):
     def buildProtocol(self, address):
         self.address = address
-        self.protocol = AccumulatingProtocol()
+        self.createdProtocol = AccumulatingProtocol()
```

Comment:
I think that we should just delete `MyOtherClientFactory` 

Is this code really used in the tests ?

Reply:
Fixed now.